templates/zerver/help/saml-authentication.md

```diff
@@ -57,6 +60,47 @@ Zulip with various common SAML identity providers.
      * Optionally you can also send us an icon that should be shown on the button.
 1. We will take care of the server-side setup and let you know as soon as it's ready.
 
+## Configure SAML with Keycloak
+
+1. Make sure you have created your organization. We'll assume its URL is
+   `https://<subdomain>.zulipchat.com` in the instructions below.
+1. Make sure your Keycloak server is up and running. We assume the URL
+   is `https://keycloak.example.com` and your Keycloak realm is `master` (which
+   is the default name for a Keycloak realm).
+1. In Keycloak, register a new Client for your Zulip organization:
+    * Client-ID: `https://<subdomain>.zulipchat.com`
+    * Client Protocol: saml
+    * Client SAML Endpoint: leave this field empty
+1. In the `Settings` tab for your new Keycloak client, set the following properties:
+    - Valid Redirect URIs: `https://<subdomain>.zulipchat.com/*`
```

Comment:
What exactly does this do? Possibly we may want the `auth` subdomain here as well

Reply:
If a user clicks `https://<subdomain>.zulipchat.com/#narrow/stream/<stream-slug>/topic/<topic.20slug>/near/<messageId>`, we'll store URL target in a `next` parameter in the SAML state, and it'll be pulled back out once authentication is complete.  The conclusion is we only need to link to `auth`.